parser-typechecker/src/Unison/Codebase/Editor/Command.hs

```diff
@@ -109,6 +110,9 @@ data Command m i v a where
   -- codebase are copied there.
   LoadLocalRootBranch :: Command m i v (Branch m)
 
+  -- Like `LoadLocalRootBranch`.
```

Comment:
Should we rename/modify `LoadLocalRootBranch` to return a `Branch.Hash` instead of a `Branch m`? and then make a helper somewhere that's `LoadLocalRootBranch >>= LoadLocalBranch`

Reply:
Actually seems like `LoadLocalRootBranch` is not used anywhere?  Unrelated to this PR, but surprising.

Reply:
This is the only one I didn't address. I guess I wasn't really convinced it mattered one way or the other.